apps/files/src/mixins.js

```diff
@@ -56,20 +56,31 @@ export default {
       if (typeof item === 'object') {
         this.selectedFile = item
         this.newName = item.name
-      } else {
-        if (item.includes('/')) {
-          this.showNotification({
-            title: this.$gettext(`Renaming of ${this.selectedFile.name} failed. The file name cannot contain a "/"`),
-            status: 'danger'
-          })
-          return
-        }
-        this.renameFile({
-          client: this.$client,
-          file: this.selectedFile,
-          newValue: item
+        return
+      }
+
+      if (item.includes('/')) {
+        this.showNotification({
+          title: this.$gettext(`Renaming of ${this.selectedFile.name} failed. The file name cannot contain a "/"`),
```

Comment:
please see #1088 on how to properly interpolate translated strings

Reply:
The interpolation then has the problem with special characters - #911

Reply:
I see ....

Reply:
$gettextInterpolate has a third argument to disable escaping - we need to see if this opens any sec issues ....

Reply:
see https://github.com/owncloud/phoenix/pull/1088/files#diff-c48186b3f0ec2249fa2211b9a5ae18beR64